Ad Astra

```diff
@@ -0,0 +1,9 @@
+{
+    "project": "Ad Astra",
+    "policies": [
+        "b8813efe62ae98fb31383e97b5cd2cf825f40ad04f4379de97540b19",
```

Comment:
Is it possible to edit the project Ad Astra, because I first created it in a way that there was no unique policy Id for the project, because I didn't know how to create it, and i did mint individual nfts and added each nft policy id to the list. I am re-doing the project as a live minting project, for which I created this unique policy ID and I won't use the pre-minted nfts (I am burning them) this would be the new project code:

{
    "project": "AdAstra",
    "tags": [
        "AdAstra"
    ],
    "policies": [
        "898982d80bc16ae205b965b723458b50dfac8695cfae93f7540c2c49"
          ]
}